src/views/layouts/teamlist.hbs

```diff
@@ -1,60 +1,149 @@
-    <section class="text-center">
-        <div class="container-fluid">
-        <h1>Teams List</h1>
-        <p>
-            <a href="/Team" class="btn btn-primary" role="button">Create New Team</a>
-        </p>
-        <div class="row justify-content-center page-divider">
-            <div class="col-auto">
-
-        <table class="table table-responsive table-bordered">
-            <thead>
-                <tr>
-                <th scope="col"></th>
-                <th scope="col">Name</th>
-                <th scope="col">Group</th>
-                <th scope="col">Games Played</th>
-                <th scope="col">Games Won</th>
-                <th scope="col">Games Lost</th>
-                <th scope="col">Games Draw</th>
-                <th scope="col">Goals</th>
-                <th scope="col">Goals Difference</th>
-                <th scope="col">Goals Group_Stage</th>
-                <th scope="col">Goals Difference Group_Stage</th>
-                <th scope="col">Sekt Won</th>
-                <th scope="col">Points Group_Stage</th>
-                <th scope="col">Points General</th>
-                <th scope="col">Actions</th>
-                </tr>
-            </thead>
-            <tbody>
-                {{#each list}}
-                <tr>
+<div class="text-center">
+    <h1 style="text-decoration: underline;">Teamverwaltung</h1>
+    <hr>
+</div>
+<div class="container">
+    <details>
+        <summary class="h2">Neues Team hinzufügen</summary>
+        <form action="/team/add" method="POST" onsubmit="return validateForm()">
+            <div class="form-group">
+                <label for="name">Name:</label>
+                <input type="text" class="form-control" id="name" name="name" value="" required>
+            </div>
+            <div class="form-group">
+                <label for="group">Gruppe auswählen oder neue Gruppe hinzufügen:</label>
+                <select class="form-control" id="group" name="group" required>
+                    <option value="Neu">Neu Gruppe:</option>
+                    {{#each groups}}
+                        <option value="{{this}}">{{this}}</option>
+                    {{/each}}
+                </select>
+                <input type="text" class="form-control mt-2" id="newGroup" name="newGroup" placeholder="Name der Neuen Gruppe eingeben" style="display: block;">
+            </div>
+            <button type="submit" class="btn btn-primary mt-3">Hinzufügen</button>
```

Comment:
Can we keep the toggle open when "Hinzufügen" is clicked? 
Would be nicer if many teams are added

Reply:
I leave it open all the time